classes/payment/omp/OMPPaymentManager.inc.php

```diff
@@ -60,8 +61,8 @@ public function createQueuedPayment($request, $type, $userId, $assocId, $amount,
 
         switch ($type) {
             case self::PAYMENT_TYPE_PURCHASE_FILE:
-                import('lib.pkp.classes.submission.SubmissionFile'); // const
-                $submissionFile = Services::get('submissionFile')->get($assocId);
+                import('lib.pkp.classes.submissionFile.SubmissionFile'); // const
```

Comment:
This `import` is not necessary since you have a `use` statement, but I won't hold back the merge on this account. (Also elsewhere)